packages/react-shortcuts/src/tests/ShortcutManager.test.tsx

```diff
@@ -235,18 +231,18 @@ describe('ShortcutManager', () => {
 
     mount(
       <ShortcutProvider>
-        <Shortcut keys={['shiftKey']} onMatch={spy} />
+        <Shortcut keys={['a']} onMatch={spy} />
```

Comment:
I guess modifier keys can't be their own shortcut any more. Makes sense, but worth noting.

Reply:
yeah, it doesn't really make sense to make them their own shortcut tbh